configs/next/packages/binutils/sources

```diff
@@ -21,7 +21,7 @@
 
 ATSRC_PACKAGE_NAME="GNU Binutils"
 ATSRC_PACKAGE_VER=2.28
```

Comment:
The version should be 2.28.51, which is read from bfd/version.m4.
However, I don't think this needs to be changed with this patch.